content/blog/2023/07/camel-k-gitops/index.md

```diff
@@ -35,15 +35,15 @@ In Camel K we use the container registry as the common place where to store the
 
 We can think on three different topologies:
 
-<img src="single_cluster.png" alt="Single cluster, shared container registry" width="560" height="774"/>
+<img src="single_cluster.png" alt="Single cluster, shared container registry" style="width:560px;height:774px;"/>
```

Comment:
Inline styling is blocked by content security policy, this will not have an effect

Reply:
Yeah, I noticed, though it does work in preview mode. Do you know which would be the proper way to set fixed width and height? I got the image wrongly resized by default otherwise.

Reply:
Replied [in chat](https://camel.zulipchat.com/#narrow/stream/257618-camel-website/topic/Image.20resize)